packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -1,579 +1,585 @@
-        // Avoid doing unnecessary work when the value has not changed.
-        if (isSameValue) {
-          return null;
-        }
-
-        // ONLY if this is an uncontrolled component, update the displayed value.
-        // (Controlled components must update the `value` prop from `onChange`.)
-        return this._isControlled ? null : { uncontrolledValue: value };
-      },
-      () => {
-        // If the value actually changed, call onChange (for either controlled or uncontrolled)
-        const { onChange } = this.props;
-        if (!isSameValue && onChange) {
-          onChange(event, value);
-        }
-      }
-    );
-  };
-
-  private _validate(value: string | undefined): void {
-    // In case _validate is called again while validation promise is executing
-    if (this._latestValidateValue === value && _shouldValidateAllChanges(this.props)) {
-      return;
-    }
-
-    this._latestValidateValue = value;
-    const onGetErrorMessage = this.props.onGetErrorMessage;
-    const result = onGetErrorMessage && onGetErrorMessage(value || '');
-
-    if (result !== undefined) {
-      if (typeof result === 'string' || !('then' in result)) {
-        this.setState({ errorMessage: result });
-        this._notifyAfterValidate(value, result);
-      } else {
-        const currentValidation: number = ++this._lastValidation;
-
-        result.then((errorMessage: string | JSX.Element) => {
-          if (currentValidation === this._lastValidation) {
-            this.setState({ errorMessage });
-          }
-          this._notifyAfterValidate(value, errorMessage);
-        });
-      }
-    } else {
-      this._notifyAfterValidate(value, '');
-    }
-  }
-
-  private _notifyAfterValidate(value: string | undefined, errorMessage: string | JSX.Element): void {
-    if (value === this.value && this.props.onNotifyValidationResult) {
-      this.props.onNotifyValidationResult(errorMessage, value);
-    }
-  }
-
-  private _adjustInputHeight(): void {
-    if (this._textElement.current && this.props.autoAdjustHeight && this.props.multiline) {
-      const textField = this._textElement.current;
-      textField.style.height = '';
-      textField.style.height = textField.scrollHeight + 'px';
-    }
-  }
-}
-
-/** Get the value from the given state and props (converting from number to string if needed) */
-function _getValue(props: ITextFieldProps, state: ITextFieldState): string | undefined {
-  const { value = state.uncontrolledValue } = props;
-  if (typeof value === 'number') {
-    // not allowed per typings, but happens anyway
-    return String(value);
-  }
-  return value;
-}
-
-/**
- * If `validateOnFocusIn` or `validateOnFocusOut` is true, validation should run **only** on that event.
- * Otherwise, validation should run on every change.
- */
-function _shouldValidateAllChanges(props: ITextFieldProps): boolean {
-  return !(props.validateOnFocusIn || props.validateOnFocusOut);
-}
+import * as React from 'react';

```

Comment:
It looks like you've somehow changed the check-in newlines in all the edited files, which needs to be reverted to avoid causing a huge amount of history noise.

How to do this varies by editor. In VS Code, towards the right of the bottom status bar, there's a little `CRLF` or `LF` indicator. Clicking that will give you the option to change the line endings. Change it to `LF` in each edited file, then save and commit.

To avoid this happening in the future, you should also configure `core.autocrlf` (either globally, or local to this repo if you prefer a different setting in general): on Windows `git config core.autocrlf true`, or on Mac `git config core.autocrlf input`, and add `-g` if you want to set globally. It's probably best to do this after fixing the existing newline issues.